packages/hooks/src/use_coroutine.rs

```diff
@@ -69,11 +69,9 @@ where
     G: FnOnce(UnboundedReceiver<M>) -> F,
     F: Future<Output = ()> + 'static,
 {
-    cx.use_hook(|| {
-        let (tx, rx) = futures_channel::mpsc::unbounded();
-        let task = cx.push_future(init(rx));
-        cx.provide_context(Coroutine { tx, task })
-    })
+    let (tx, rx) = futures_channel::mpsc::unbounded();
+    let task = cx.push_future(init(rx));
```

Comment:
I think this should still be in the use_hook. We only want to create and spawn the coroutine future when the component is first created, not every time the component is rerun

Reply:
Forgot to change it back, thanks!